bindings/python/iio.py

```diff
@@ -437,6 +476,16 @@ def _Attr__write(self, value):
 
 class Channel(object):
 	def __init__(self, _channel):
```

Comment:
Need a docstring above the init like the others.

Reply:
Solved.